src/common/BrokerDetailsBreadcrumb.tsx

```diff
@@ -0,0 +1,27 @@
+import { Breadcrumb, BreadcrumbItem } from '@patternfly/react-core';
+import { useTranslation } from '../i18n';
+
+const BrokerContainerBreadcrumb = (props: { currentName: string }) => {
+  const { t } = useTranslation();
+  const addBrokerName = () => {
+    if (props.currentName) {
+      return (
+        <BreadcrumbItem to="/k8s/all-namespaces/brokers/:name" isActive>
```

Comment:
namespace should be dynamic. It  should be come as prop from details page similar to name.

```suggestion
        <BreadcrumbItem to=`/k8s/$/{namespace}/brokers` isActive>
```